recipes/qt/6.x.x/conanfile.py

```diff
@@ -467,7 +467,7 @@ def build(self):
             tools.replace_in_file(f,
                 "$<$<STREQUAL:$<TARGET_PROPERTY:TYPE>,MODULE_LIBRARY>:-Wl,--export-dynamic>",
                 "", strict=False)
-        with tools.vcvars(self.settings) if self.settings.compiler == "Visual Studio" else tools.no_op():
+        with tools.vcvars(self.settings, winsdk_version="10.0.18362.0") if self.settings.compiler == "Visual Studio" else tools.no_op():
```

Comment:
Won't this break with every single small update in CI and every small difference in the version locally?

Reply:
Yes, this was a test to check if the sdk was installed